handler.cpp

```diff
@@ -5,33 +5,9 @@
 #include <vector>
 #include <string>
 
-echo_handler::echo_handler(std::vector<char>& request) {
```

Comment:
yeah you're right, however its in the commit history. I am working on implementing this right now anyway so no worries.